apps/gitness/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "gitness",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
```

Comment:
why do we need a `tsc` build here?

Reply:
in fact we should add `build:css` and change this to `build:css && vite build`

Reply:
Cleaned this up. Added `"build": "vite build && pnpm build:css"`